pipeline/outputs/s3.md

```diff
@@ -282,6 +290,44 @@ Example:
 
 Then, the records will be stored into the MinIO server.
 
+## Usage for Parquet Compression
+
+For parquet compression, users need to install to install [columnify](https://github.com/reproio/columnify) in the running system or container at runtime.
+
+After installing that command, out_s3 can handle parquet compression:
+
+```
+[OUTPUT]
+    Name  s3
+    Match *
+    bucket your-bucket
+    Use_Put_object true
+    compression parquet
+    parquet.schema_file /path/to/your-schema.avsc
+    parquet.compression snappy
+```
+
+### Build columnify Command
+
+For building columnify command, users need to inherit and use Golang development container:
+
+```
+FROM golang:1-alpine as builder # Always refer the latest golang:1-aline image
+
+ENV ROOT=/go/src/cmd
+WORKDIR ${ROOT}
+
+RUN apk update && \
+  apk add git
+
+RUN go install github.com/reproio/columnify/cmd/columnify@latest
+
+FROM debian:bullseye-slim as production
+
+# Put columnify command inside the PATH.
+COPY --from=builder /go/bin/columnify /usr/bin/columnify
```

Comment:
Does the Dockerfile need a CMD or ENTRYPOINT?

Reply:
I think that it's not needed because it's a just example for building for columnify command.